### PR TITLE
virtio_fs:  virtiofs code update/add after including functions in provided unit

### PR DESCRIPTION
**Pull request by xiagao**
1.For virtiofs service test, it's better to clear the virtiofs service at the end. For virtiofs device hotplug/unplug test, deleting virtiofs service will fail after unplugging the virtiofs device, so use the image backend method for it.
2. Do some code update when using functions in provided unit
    1.Update parameters as creating/deleting virtiofs
    service is using function unit that implemented
    in provider/win_driver_installer_test.py instead.
    2.Use stop virtiofs service instead of delete service
    when creating multi virtiofs service

ID: 2156433, 2169907
Signed-off-by: Xiaoling Gao <xiagao@redhat.com>

### CODE REVIEW

**Comment by xiagao** [comment]
@leidwang could you pls review this patch,thanks.

**Review by leidwang — DISMISSED** [review]
LGTM.

**Comment by leidwang** [comment]
Hi @xiagao Could you paste the test results here?

**Comment by xiagao** [comment]
> Hi @xiagao Could you paste the test results here?

sure, I submit a test loop to verify it, it's still running.

**Comment by xiagao** [comment]
@leidwang  The loop result is good, update it in bugzilla.

**Comment by xiagao** [comment]
Now I will resolve the conflict.

**Comment by leidwang** [comment]
> @leidwang The loop result is good, update it in bugzilla.

BZ2156433 ?

**Comment by xiagao** [comment]
@leidwang could you have a check? The rebase conflict is resolved.

**Review by leidwang — APPROVED** [review]

**Comment by xiagao** [comment]
@YongxueHong  Could you help review it?

File: qemu/tests/virtio_fs_share_data.py [hunk 1]
```diff
@@ -811,6 +810,11 @@ def check_security_label(file, folder, xattr_name):
         if os_type == "windows":
             win_driver_utils.memory_leak_check(vm, test, params)
     finally:
+        if os_type == 'windows' and vm and vm.is_alive():
```
Comment (YongxueHong) [inline qemu/tests/virtio_fs_share_data.py:813]: Hi @xiagao 
I think just this part relates to the commit message, but other changes do not. 
So I don't understand the purpose of those changes.
Could you split those changes into different commits?

Comment (xiagao) [reply qemu/tests/virtio_fs_share_data.py:813]: Hi @YongxueHong 
I update the commit, for the other code, most of them are parameters update because we use some unit function in https://github.com/autotest/tp-qemu/pull/3596, some parameters are inconsistent.

**Comment by xiagao** [comment]
The loop results are good and they were updated in bz.

**Review by YongxueHong — APPROVED** [review]
LGTM.

**Review by leidwang — APPROVED** [review]
